permits/admin.py

```diff
@@ -364,11 +380,45 @@ def clean_permissions(self):
 class GroupAdmin(admin.ModelAdmin):
     inlines = (PermitDepartmentInline,)
     form = GroupAdminForm
+    list_display = [
+        "__str__",
+        "get__is_validator",
+        "get__is_default_validator",
+        "get__integrator",
+        "get__mandatory_2fa",
+    ]
+
```

Comment:
If not too complicated, change the display to checkbox like in WOTs for instance (1.4)

![image](https://user-images.githubusercontent.com/3356536/154935597-2219bf00-ab24-4526-a525-06ba4fae0b8d.png)